README.md

```diff
@@ -8,7 +8,7 @@
 1. `bundle install`
 1. Generate a secret key with `rake secret` and paste this value into the `application.yml`.
 1. Run `yarn` to install webpack needs
-1. `rails db:create db:migrate`
+1. `rails db:create db:migrate` FEEDBACK: You can just do `rails db:setup` and it will create + migrate + seed
```

Comment:
agree, that option can be added to the readme. sometimes people write seeds that can't be ran twice, so it may be discussable sometimes imo